env_spec.py

```diff
@@ -0,0 +1,90 @@
+spec_str = 'DATABASE_URL\nADMIN_EMAIL\nDEBUG'
+
+
+def str_split(spec_str):
+    """
+    split input string by \n and returns strings into an array
+    """
+    field_array = []
```

Comment:
This is redundant, `field_array` will be initialised automatically from `spec_str.split...`.